Get Started/landing.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="https://endlessui.netlify.app/css/intro.css" />
+    <link rel="stylesheet" href="/Get Started/landing.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home Page</title>
+  </head>
+  <body>
+    <nav>
+      <div class="nav-bar">
+          <div class="left-navbar">
+            <h2><a href="/Get Started/landing.html">    <span class="primary-highlight">Untz</span>Untz!</a></h2>
+          </div>
+        <div class="right-navbar">
+          <a href="/Pages/login_signup/login.html">Login</a>
+          <a href="/Pages/login_signup/signup.html">Signup</a>
+        </div>
+      </div>
+    </nav>
+    <div class="landing-content">
+        <div class="landing-text">
+            <p>Your Neighbors Will Listen To Good Music Whether They Like It Or Not</p>
+            <button class="btn-getstarted">
+                <a href="/Pages/login_signup/signup.html">Get Started</a>
+            </button>
+        </div>
+        <div class="landing-image">
+            <img src="/Images/undraw_mello_otq1.svg" alt="">
```

Comment:
Here you have added alt but it should add information about the image. because if your img fails to load user can see this alt on screen and having a meaningful alt will help.